File: kotlin/task2.kt
fun countDigits(num: Int): Int = if (num == 0) 1 else num.toString().length

fun main() {
    print("Введите числа через пробел: ")
    val numbers = readLine()!!.split(" ").map { it.toInt() }

    val count = numbers.count { countDigits(it) % 2 != 0 }
    println("Количество чисел с нечётным числом цифр: $count")
}

File: c#/task2.cs
using System;
using System.Linq;

class Task2 {
    static int CountDigits(int n) {
        return n.ToString().Length;
    }

    static void Main() {
        Console.Write("Введите числа через пробел: ");
        var numbers = Console.ReadLine().Split().Select(int.Parse).ToList();

        int count = numbers.Count(n => CountDigits(Math.Abs(n)) % 2 != 0);
        Console.WriteLine("Количество чисел с нечётным количеством цифр: " + count);
    }
}
